sympy/printing/latex.py

```diff
@@ -2404,6 +2404,21 @@ def _print_Morphism(self, morphism):
         codomain = self._print(morphism.codomain)
         return "%s\\rightarrow %s" % (domain, codomain)
 
+    def _print_TransferFunction(self, expr):
+        from sympy.core.mul import Mul
+        num, den = expr.num, expr.den
+        res = Mul(num/den, evaluate=False)
+        return self._print_Mul(res)
```

Comment:
Using `Mul(num/den, evaluate=False)` doesn't prevent evaluation because `num/den` evaluates before being passed to `Mul`. Since you only pass a single arg to `Mul` it will simply return that argument unchanged (even if `evaluate=False`). If you wanted to prevent evaluation you could use e.g.:
```
res = Mul(num, Pow(den, -1, evaluate=False), evaluate=False)
```

Reply:
@namannimmo10 Do you have a unit test that ensures this does what is expected? If not you need one.

Reply:
@oscarbenjamin, Thanks for pointing this, I totally missed it! Yes, I will update the unit tests.